setup.py

```diff
@@ -117,7 +117,7 @@
             # For spelling
             "sphinxcontrib.spelling",
             # Type hints support
-            "sphinx-autodoc-typehints",
+            "sphinx-autodoc-typehints==1.21.1",  # TODO: remove version constraint, see #1290
```

Comment:
after a quick check, gym 0.26 doesn't solve that issue... so I guess best is to override the docstrings (as long as there are not too many)

Reply:
Wonder if it's not simply a bug in `sphinx-autodoc-typehints`.

Reply:
looks like so